Signal/src/environment/NotificationsManager.m

```diff
@@ -155,6 +154,11 @@ - (void)notifyUserForIncomingMessage:(TSIncomingMessage *)message
     }
 
     NSString *messageDescription = message.description;
+    NSString *senderName = [contactsManager displayNameForPhoneIdentifier:message.authorId];
+    NSString *groupName = [thread.name stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceCharacterSet]];
```

Comment:
This name is a little confusing, because it will be populated with the contact's name for ContactThreads, right? 

Since we never want dangling white space in a thread name, maybe this whitespace trimming belongs at a higher level, since apparently we're not doing a good job of trimming group names when their set.